libPhoneNumber/NBGeocoderMetadataHelper.h

```diff
@@ -0,0 +1,27 @@
+//
+//  NBGeocoderMetadataHelper.h
+//  libPhoneNumberiOS
+//
+//  Created by Rastaar Haghi on 6/12/20.
+//  Copyright © 2020 ohtalk.me. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import <sqlite3.h>
+#import "NBPhoneNumber.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface NBGeocoderMetadataHelper : NSObject
+
+@property(nonatomic, copy) NSString *regionDescription;
```

Comment:
What is this property for?

Reply:
was to store most recent description in class, but realizing now that it is unnecessary.